create_subgraph.py

```diff
@@ -100,7 +125,15 @@ def user_defined_edge_exclusion(graph,kg_type ):
     return(graph)
 
 
-
+# Edges to remove
+def automatic_defined_edge_exclusion(graph,kg_type):
```

Comment:
This doesn't seem to be working. Should remove the defined edges (ie., "only_in_taxon" which is <http://purl.obolibrary.org/obo/RO_0002160> but this is still included in the resulting patterns.